src/Adapter/Product/CommandHandler/DeleteProductHandler.php

```diff
@@ -28,24 +28,29 @@
 
 namespace PrestaShop\PrestaShop\Adapter\Product\CommandHandler;
 
-use PrestaShop\PrestaShop\Adapter\Product\Repository\ProductRepository;
+use PrestaShop\PrestaShop\Adapter\Product\Repository\ProductMultiShopRepository;
 use PrestaShop\PrestaShop\Core\Domain\Product\Command\DeleteProductCommand;
 use PrestaShop\PrestaShop\Core\Domain\Product\CommandHandler\DeleteProductHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\Shop\ValueObject\ShopConstraint;
 
 /**
  * Handles @see DeleteProductCommand using legacy object model
+ *
+ * @todo: double check this command and DeleteProductFromShops. Maybe only one of them should exist?
+ *        Or maybe this one needs to be renamed if its used for all shops?
+ *        Previous behavior of this command was probably to remove product from current context shop only
```

Comment:
ping @jolelievre, you introduced DeleteProductFromShops command recently, buy I think its not used, maybe you can help me sort it out how these 2 commands should behave now?